src/mysql.c

```diff
@@ -995,6 +1024,54 @@ static int mysql_read (user_data_t *ud)
 		{
 			counter_submit ("mysql_slow_queries", NULL , val, db);
 		}
+ 		else if (strncmp (key, "Queries", strlen ("Queries")) == 0) 
+ 		{
+ 			counter_submit ("mysql_queries", NULL , val, db);
+ 		}
+ 		else if (strncmp (key, "Connections", strlen ("Connections")) == 0) 
+ 		{
+ 			counter_submit ("mysql_connections", NULL , val, db);
+ 		}
+ 		else if (strncmp (key, "Uptime", strlen ("Uptime")) == 0) 
+ 		{
+ 			counter_submit ("mysql_uptime", NULL , val, db);
```

Comment:
What is the sense of this metric? When I want to check uptime I refer to nagios, there I can found all required history of status changes.  As for me, I see nothing useful in triangles on similar charts.